elliottlib/bzutil.py

```diff
@@ -22,6 +22,8 @@
 from elliottlib.cli import cli_opts
 from elliottlib.metadata import Metadata
 from elliottlib.util import isolate_timestamp_in_release
+from errata_tool.jira_issue import JiraIssue
```

Comment:
Maybe give this another name that tells that it comes from Errata world. `ErrataJira` maybe?